app/lib/frontend/handlers.dart

```diff
@@ -446,19 +435,30 @@ Future<shelf.Response> _packagesHandlerHtmlV2(
     shelf.Request request, String platform) async {
   final int page = _pageFromUrl(request.url);
   final int offset = PackageLinks.resultsPerPage * (page - 1);
+  final queryText = request.requestedUri.queryParameters['q'] ?? '';
+  // TODO: handle package-prefix
+
+  // TODO: handle sort order
```

Comment:
Yeah, this makes this pretty broken

Reply:
Yeah, the default order is context-dependent.